From 598580cf7374ddff2216428a7b76b91c1e64ff39 Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Sun, 17 Jan 2016 17:51:22 +0100
Subject: [PATCH] Bump device name length to 32

Some 3rd party kernel modules create devices which has names longer than
16 characters.

New length 32 was picked pretty much arbitrary. However, my decision was
reinforced when I checked Debian's ConsoleKit package and found out they
already ship the same patch.

Resolves: #1070249
---
 src/ck-sysdeps-linux.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ck-sysdeps-linux.c b/src/ck-sysdeps-linux.c
index 09db310..7a09ee7 100644
--- a/src/ck-sysdeps-linux.c
+++ b/src/ck-sysdeps-linux.c
@@ -93,7 +93,7 @@ typedef struct tty_map_node {
         guint major_number;
         guint minor_first;
         guint minor_last;
-        char name[16];
+        char name[32];
         char devfs_type;
 } tty_map_node;
 
-- 
2.5.0

